### DISCUSSION TITLE
New ecommerce built with Saleor!

### DISCUSSION THREAD

**Discussion by DidoMarchet** [saleor/saleor#11994] [Show and tell]
We're excited to share the new online store we developed using Saleor. I really enjoyed creating its architecture and related integrations.
One of the best things about Saleor is that it's easy to integrate. We were able to develop our own custom checkout and dashboard, which we can also use in future projects, saving us a lot of time.

To enhance the product page with more content from DATO CMS, we used  plugin I developed  plugin I developer https://github.com/DidoMarchet/datocms-plugin-saleor. 

You can see the results for yourself by visiting https://gopleyers.com/. 
Soon, the website will be available in English too! :D

Have a nice day, Davide

**Comment by karolkielecki** [comment]
Excellent job on the project! Thanks for sharing, @DidoMarchet!
